flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/StreamTableEnvironment.scala

```diff
@@ -667,30 +719,62 @@ abstract class StreamTableEnvironment(
     // get CRow plan
     val plan: DataStream[CRow] = translateToCRow(logicalPlan, queryConfig)
 
+    val rowtimeFields = logicalType
+      .getFieldList.asScala
+      .filter(f => FlinkTypeFactory.isRowtimeIndicatorType(f.getType))
+
+    // convert the input type for the conversion mapper
+    // the input will be changed in the OutputRowtimeProcessFunction later
+    val convType = if (rowtimeFields.size > 1) {
+      throw new TableException(
```

Comment:
This is a very good question. I don't know how we want to solve the `SELECT *` problem. My initial idea was to just the attribute of the left table as a time indicator. Another idea would be to remove all time indicators and explicitly choose one attribute `SELECT TIME(OrderA.rowtime)`. Still not perfect. We need to discuss this. @fhueske what do you think?